### DISCUSSION TITLE
How to do parameter identification and optimization, sensitivity analysis?

### DISCUSSION THREAD

**Discussion by chzhcpu** [nlmixr2/nlmixr2#36] [General]
Many parameters are required in the PBPK model, and some parameters have no experimental data. Can this software perform parameter identification and optimization, as well as sensitivity analysis? Are there any examples? thks.

**Comment by mattfidler** [comment]
Hi,  

- `nlmixr2` can currently optimize for the best parameters
- `nlmixr2` cannot currently determine if the parameter is identifiable
- `nlmixr2` currently doesn't have a convivence function for sensitivity analysis.

**Comment by mattfidler** [comment]
If you manage to write a generalizable method for this, please contribute back.

**Comment by chzhcpu** [comment]
@mattfidler 
nlmixr2 is a good piece of software and I'm learning how to use it. 
But I'm not a programmer and can't program.

**Comment by mattfidler** [comment]
Ok,  I do understand.  We do not have this functionality currently
